gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -189,6 +192,43 @@ def call_reset(self, _):
         return MENU_RESET
 
     # COMMANDS
+    @try_except
+    def call_search(self, other_args: List[str]):
+        """Process search command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="search",
+            description="Show companies matching the search query.",
+        )
+
+        parser.add_argument(
+            "-q",
+            "--query",
+            action="store",
+            dest="query",
+            type=str.lower,
+            required="-h" not in other_args,
+            help="The search term used to find company tickers.",
+        )
+
+        parser.add_argument(
+            "-a",
+            "--amount",
+            default=10,
+            type=int,
+            dest="amount",
+            help="Enter the number of Equities you wish to see in the Tabulate window.",
+        )
+
+        if other_args and "-q" not in other_args and "-h" not in other_args:
+            other_args.insert(0, "-q")
+
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+
```

Comment:
Should include `if not ns_parser: return`

That way doing -h doesn't show the error that passes through

Reply:
Done!